paddle/gserver/dataproviders/DataProvider.h

```diff
@@ -349,7 +351,6 @@ class DataProvider {
    */
   virtual void reset() {
     if (doubleBuffer_ != nullptr) {
-      LOG(INFO) << "the double-buffer is starting ...";
```

Comment:
Because PyDataProvider2 is always using double-buffer, and PyDataProvider2 is the only interface to providing data in opensource.  So remove the noising line here.